src/main/kotlin/com/cosmotech/api/config/CsmPlatformProperties.kt

```diff
@@ -449,6 +449,9 @@ data class CsmPlatformProperties(
 
       /** TLS Platform bundle config */
       val tls: TLSConfig = TLSConfig(),
+
+      /** Use Graph module */
+      val useGraphModule: Boolean = true
```

Comment:
Not sure if this is worth modifying, but did you consider something like `graphModule.enabled` with a "group"? That may simplify things to already have a dedicated section if we need to extend that part of the config.

Reply:
I thought to do that also but as right now it's just a way to active/deactivate existing graph functionalities and not a way to handle a dedicated graph solution, I choose to leave it like this.
In 4.0-GA, it will certainly be the way that we will manage it but even under a new property as graph solution will be not part of twincache anymore.